InformationScripting/src/dataformat/TupleSet.h

```diff
@@ -57,6 +55,12 @@ class INFORMATIONSCRIPTING_API TupleSet
 		QSet<Tuple> takeAll();
 		void unite(const TupleSet& with);
 
+		/**
+		 * Adds all properties of type \a T as single Tuples with tag \a tag.
+		 */
+		template<class T>
+		void addAllPropertiesTo(const QString& tag);
```

Comment:
I'm not quite happy with the name of this method. Let's call it something more descriptive, such as `addPropertiesAsTuples` or some other better name.